src/translations/locales/common_es.json

```diff
@@ -62,7 +62,7 @@
       "howToBook": {
         "title": "Cómo reservar",
         "stepOne": "Encuentre una ubicación y hora en la lista a continuación.",
-        "stepTwo": "Clickea en el <i>Make a Booking</i> botón, esto te llevará a <0>bookmyvaccine.nz</0>",
+        "stepTwo": "Clickea en el <i>hacer una reserva</i> botón, esto te llevará a <0>bookmyvaccine.nz</0>",
```

Comment:
It should be "Clickea en el botón <i>hacer una reserva</i>, esto te llevará a <0>bookmyvaccine.nz</0>"